cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -462,6 +450,21 @@ body {
                 padding-right: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
             }
         }
+
+        // Group heading - Tablet/Mobile
+        &_group-heading {
+            margin-top: @grid_gutter-width;
+
+            &__hidden {
+                .u-visually-hidden;
+            }
+
+            // add border between title and following list
+            &:not(&__hidden) + ul {
+                border-top: 1px solid @gray-40;
+            }
+        }
+
     } );
 
     // Desktop sizes.
```

Comment:
Missed an update to the `.nav_footer` class name below on L474.

Reply:
Thanks for catching that! Should be addressed in https://github.com/cfpb/cfgov-refresh/pull/3280/commits/08dcff0f6b54c7ab654a9e67d0fcb5ea126d9ddb